src/lib/registration/ui/migration_repos_workflow.rb

```diff
@@ -753,6 +764,20 @@ def unregistered_online_message
           _("<p>Another option is booting the original system and registering it.</p>")
       end
 
+      # Informative message
+      # @return [String] translated message
+      def online_media_upgrade
+        # TRANSLATORS: Media upgrade forced but the Online medium was detected (1/2)
+        #   Message displayed during upgrade when the Online medium is used
+        #   with the "media_upgrade=1" boot option. The Online medium does not
+        #   contain any packages and it cannot be used in this case, Full medium
+        #   is required. Use the RichText format.
+        _("<h2>Online Medium</h2><p>The system cannot be upgraded using the Online " \
```

Comment:
I am not sure whether the message is clear enough. I would indicate the source of the problem (i.e., because the media based upgrade was requested). I suggest something like:

"The media based upgrade was requested, but you are using the Online medium which does not provide any packages to install. Please use the Full medium instead of the Online."

What do you think?

Reply:
suggestion sound fine for me, I just do not like that direct "you". Better impersonate with "but the Online medium is used"

Reply:
What about "The media based upgrade was requested, but the Online medium is used. The Online medium does not provide any packages to install. Please use the Full medium instead of the Online."